napari/_app_model/constants/_menus.py

```diff
@@ -62,14 +106,18 @@ class LAYERLIST_CONTEXT:
         SPLIT_MERGE = '5_split_merge'
         LINK = '9_link'
 
-
-# TODO: add these to docs, with a lookup for what each menu is/does.
-_CONTRIBUTABLES = {MenuId.LAYERLIST_CONTEXT.value}
-"""Set of all menu ids that can be contributed to by plugins."""
+    class LAYERS:
+        CONVERT = '1_convert'
+        GEOMETRY = '2_geometry'
+        GENERATE = '3_generate'
```

Comment:
<img width="172" alt="Screenshot 2024-06-26 at 6 38 25 PM" src="https://github.com/napari/napari/assets/492549/afc5c67e-d723-412f-8cfc-a0edf02e1744">

Do these group names still make sense? (Is that what these are?)